css/style.css

```diff
@@ -104,3 +121,530 @@ body {
+  border: 1px  solid #396df2;
+  padding: 8px;
+  border-radius: 10px;
+  display: flex;
+  text-align: center;
+  align-items: center;
+  letter-spacing: 0.03em;
+  background-color: #fff;
+  margin-left: 16px;
+  margin-top: 30px;
+}
+
+.fourth_card {
+  width: 327px;
+  height: 596px;
+  background-color: #fff;
+  margin: 116px auto;
+  border-radius: 16px;
+}
+
+.fourth_card img {
+  padding-top: 16px;
+  width: 295px;
+  height: 220px;
+  margin-left: 16px;
+}
+
+.fourth_card h2 {
+  width: 295px;
+  height: 36px;
+  margin-left: 16px;
+  font-family: 'poppins', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  color: #172b4d;
+  margin-top: 8px;
+}
+
+.fourth_card h3 {
+  width: 55px;
+  height: 16px;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  font-family: 'poppins', sans-serif;
+  color: #344563;
+  margin-left: 16px;
+  margin-top: 70px;
+}
+
+.fourth_card .birth {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  display: flex;
+  justify-content: space-evenly;
+  margin-left: -12px;
+  margin-top: -28px;
+}
+
+.fourth_card .year {
+  margin-left: -45px;
+}
+
+.fourth_card p {
+  width: 295px;
+  height: 76px;
+  font-family: 'poppins1', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 20px;
+  margin-left: 16px;
+}
+
+.fourth_card .button li {
+  font-size: 10px;
+  font-weight: 500;
+  line-height: 16px;
+  list-style-type: none;
+  padding: 4px 12px;
+  margin: auto;
+  border-radius: 10px;
+  background-color: #ebebff;
+  color: #6070ff;
+  display: inline-block;
+}
+
+.fourth_card button {
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  color: #396df2;
+  border: 1px  solid #396df2;
+  padding: 8px;
+  border-radius: 10px;
+  display: flex;
+  text-align: center;
+  align-items: center;
+  letter-spacing: 0.03em;
+  background-color: #fff;
+  margin-left: 16px;
+  margin-top: 20px;
+}
+
+.about {
+  width: 375px;
+  height: 1250px;
+  border-radius: 0 80px 0 0;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+}
+
+.about h1 {
+  width: 327px;
+  height: 35px;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  font-family: 'poppins', sans-serif;
+  color: #172b4d;
+  margin-top: 100px;
+  margin-left: 16px;
+}
+
+.about p {
+  width: 327px;
+  height: 147px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  font-family: 'poppins1', sans-serif;
+  margin-left: 16px;
+  margin-top: -20px;
+}
+
+.about span {
+  width: 117px;
+  height: 24px;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  font-family: 'poppins1', sans-serif;
+  color: #7f8cff;
+  margin-left: 16px;
+}
+
+.about .social_media li {
+  list-style-type: none;
+  display: inline-block;
+  padding: 0 7px;
+  position: relative;
+  left: -20px;
+}
+
+.about .social_media .media {
+  color: #505f79;
+  font-size: 25px;
+}
+
+.about button {
```

Comment:
- [ ] I kindly suggest that you add the button interactions for active (when it's pressed) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on hover has the  `background-color: #2230D2;` 
### EXAMPLE:
```
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```